Project/Project/Pages/Index.cshtml

```diff
@@ -11,8 +11,10 @@
     <div class="col-md-4">
         <form method="get">
             <div class="form-group">
-                <label class="control-label"></label>
-                <select asp-for="Query" class="form-control" asp-items="ViewBag.Code"></select>
+                @*<label class="control-label"></label>*@
+                <select asp-for="Query" class="form-control" asp-items="ViewBag.Code">
+                    <option value="">Select the counrty to show Nobel Laureate</option>
```

Comment:
The content between <option> tag can replace the <lable> for better HTML displaying.